### ISSUE TITLE
On the server side, does jsmpp validate src/dest addresses based on TON and NPI?

### ISSUE THREAD

**Issue by javadevmtl** [opentelecoms-org/jsmpp#143]
As the title asks. Is there validation or the values are provided as is?

So if a client connects to our smpp server implemented with Jsmpp and they send a number that does not match against TON/NPI will it get rejected or we have to implement the validation?

**Comment by pmoerenhout** [comment]
Yes, you have to implement some logic to decide what you want to accept. See the WaitBindTask in the examples.
